docs/enterprise/sso.md

```diff
@@ -0,0 +1,402 @@
+---
+layout: default
+title: Single Sign On (SSO)
+description: This section covers authentication (using SSO) of your lakeFS Cloud or lakeFS Enterprise.
+parent: lakeFS Enterprise
+nav_order: 65
+has_children: false
+---
+
+# Single Sign On (SSO)
+{: .d-inline-block }
+
+lakeFS Enterprise
+{: .label .label-purple }
+{: .no_toc }
+
+
+{: .note}
+> SSO is only available for [lakeFS Cloud](../cloud/) and [lakeFS Enterprise](../enterprise/).
+>
+> Using the Open Source? Read more on [authentication](authentication.html). 
+
+{% include toc.html %}
+
+## Supported Protocols and Third Parties
+
+lakeFS Cloud and lakeFS Enterprise support the commonly used protocols for authentication, such as OpenID Connect and SAML.
+
+There are specific authentication utilities that we've already verified that are working, if you're using one that is not listed below but is based on OpenID Connect or SAML, it's probably a small effort for us to make sure it'll work, please [contact us](support@treeverse.io) for more information.
+
+
+## Enterprise
+
+lakeFS Enterprise provides a secondary service, running side-by-side with lakeFS which handles the authentication, this service is called Fluffy.
+Once you've onboarded to [lakeFS Enterprise](../enterprise.md) and have access to the authentication service (Fluffy), you can configure one of the supported authentication methods below.
+lakeFS Enterprise leverage Helm's capabilities to run the sidecar with it's configuration, on top of the specific-authentication method, please review the [Helm configuration](#helm).
+
```

Comment:
Also, I've left this in but don't understand what it means in the context of this document - is it important to include in the introduction or is it self-evident from the instructions that the user follows? If the former can it perhaps be reworded to be clearer please? 

> lakeFS Enterprise uses Helm's capabilities to run the sidecar with its configuration, on top of the specific authentication method, please review the [Helm configuration](#helm).

Reply:
should this just be moved to the Helm section itself? Or just removed?

Reply:
it's important to use it with Helm, this configuration is specific for Helm right now and the best practice for deploying this server for lakeFS Enterprise.
Once we'll have the secondary helm chart, it can be removed.